src/main/frontend/components/block.cljs

```diff
@@ -2774,15 +2774,21 @@
       (when-not (and built-in? (empty? result))
         [:div.custom-query.mt-4 (get config :attr {})
          (ui/foldable
-          [:div.custom-query-title
-           [:span.title-text (cond
+          [:div.custom-query-title.flex.justify-between.w-full
+           [:div [:span.title-text (cond
                                (vector? title) title
                                (string? title) (inline-text config
                                                             (get-in config [:block :block/format] :markdown)
                                                             title)
                                :else title)]
            [:span.opacity-60.text-sm.ml-2.results-count
             (str (count transformed-query-result) " results")]]
+           [:a.opacity-70.hover:opacity-100.svg-small.inline ;add an edit button
```

Comment:
For a comment that refers to multiple lines of code or a fragment of code, it's more common to put above it https://github.com/bbatsov/clojure-style-guide#code-fragment-line-comments . We should probably refer to that guide in our dev docs soon as that is the style we turn to the most

Reply:
@logseq-cldwalker I think I know what I'll be reading tonight!

I've pushed a commit which I believe ensures compliance.